management/database/entities.go

```diff
@@ -0,0 +1,67 @@
+package database
+
+import (
+	"encoding/xml"
+)
+
+// DatabaseServerCreateParams represents the set of possible parameters
+// when issuing a database server creation request to Azure.
+//
+// https://msdn.microsoft.com/en-us/library/azure/dn505699.aspx
+type DatabaseServerCreateParams struct {
+	XMLName  xml.Name `xml:"http://schemas.microsoft.com/sqlazure/2010/12/ Server"`
+	UserName string   `xml:"AdministratorLogin"`
+	Password string   `xml:"AdministratorLoginPassword"`
+	Location string
+	Version  string
+}
+
+// DatabaseServerCreateResponse represents the response following the creation of
+// a database server on Azure.
+type DatabaseServerCreateResponse struct {
+	ServerName string
+}
+
+const (
+	DatabaseServerVersion11 = "2.0"
+	DatabaseServerVersion12 = "12.0"
+)
+
+// DatabaseServerResponse represents the set of data recieved from
+// a database server list operation.
+//
+// https://msdn.microsoft.com/en-us/library/azure/dn505702.aspx
+type DatabaseServer struct {
+	Name     string
+	UserName string `xml:"AdministratorLogin"`
+	Location string
+	DNSName  string `xml:"FullyQualifiedDomainName"`
+	Version  string
+	State    string
+}
+
+type ListDatabaseServersResponse struct {
+	XMLName xml.Name         `xml:"Servers"`
+	Servers []DatabaseServer `xml:"Server"`
+}
+
+// Database represents the set of possible parameters when issuing
+// a database creation to Azure, and reading a list response from Azure.
```

Comment:
please do not reuse the same type `Database` for both serializing to requests and deserializing from responses. you can alternatively embed this type (make it unexported) into two exported types with xxxRequest and yyyResponse.